src/main/java/io/strimzi/kafka/metrics/PrometheusMetricsReporterConfig.java

```diff
@@ -107,6 +133,10 @@ public String toString() {
      */
     public synchronized Optional<HTTPServer> startHttpServer() {
         try {
+            if (!listenerEnabled) {
```

Comment:
This can be moved out of the `try` block

Reply:
Moved out of `try` block